src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -1783,6 +1784,7 @@ bool CacheManager::Init(const std::string& file_name,
       LOG4CXX_INFO(logger_, "Policy Table was inited successfully");
 
       result = LoadFromFile(file_name, *pt_);
+      UnwrapAppPolicies(pt_->policy_table.app_policies_section.apps);
```

Comment:
`UnwrapAppPolicies` is bool so it may fail. 
Good practice - to check return results. 

I would propose putting `UnwrapAppPolicies` after `snapshot->is_valid();`. 
And make the `result` depending from `UnwrapAppPolicies` : `result &= UnwrapAppPolicies(...`

Also, looks like we need to add additional Unit test for Init Function, that will check that is actually does unwrapping policies.

Reply:
1. Updated test script is attached to original issue.
2. Unfortunately, class CacheManager isn't covered by unit tests at all, this problem is reflected in issue # 2449.
3. Fix for external flow is added in commit 4c96187dc1b2f5f8bab7b820dfdf9d5713ba136f.
4. Full regression is passed successfully.